ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1393 @@
+{
```

Comment:
<p>You don't actually compare those results with the ones you had in training phase.</p><p> </p><p>But why do you want to do this? When do you expect to get results that differ from the ones you got in training?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>There are some problem with save all outputs in JupyterNotebooks.</p><p>I have some idea how to show the training result in notebooks.</p>